pulp_smash/tests/platform/api_v2/simple_repo_test.py

```diff
@@ -0,0 +1,174 @@
+# coding=utf-8
+"""Test for basic repo creating functionality.
+"""
+from __future__ import unicode_literals
+
+from pulp_smash.resources.platform.api_v2.api import Repository, Task
+from pulp_smash.resources.platform.api_v2.api import ERROR_KEYS
+from unittest2 import TestCase
+
+
+class RepoCreateSuccessTestCase(TestCase):
+    """Tests for successfull repo creating functionality.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create repo on pulp server
+        """
+        cls.repo = Repository(id=cls.__name__)
+        cls.repo.create_repo()
+
+    def test_status_code(self):
+        """Test if Create repo returned 201."""
+        self.assertEqual(self.repo.last_response.status_code, 201)
+
+    def test_correct_id(self):
+        """Test if response contain correct repo id.
+        """
+        self.assertEqual(
+            self.repo.last_response.json()['id'],
+            self.__class__.__name__,
+            set(self.repo.last_response.json()))
```

Comment:
pep8 allows for quite a bit of flexibility in formatting, and there are no really great code formatters for Python as in other languages, like Go and C. As a result, many different styles have emerged.

Please use the following layout for all method and function calls unless the situation strongly calls for something different. I know it's nit-picky of me to ask for this. But this layout is formulaic enough to work nearly everywhere, and using a consistent style makes it possible for people to easily scan through code.

```python
# Yes.
self.assertEqual(
    self.repo.last_response.json()['id'],
    self.__class__.__name__,
    set(self.repo.last_response.json())
)
self.assertEqual(self.repo.last_response.json()['id'], self.__class__.__name__)

# No: closing parenthesis is not at same indentation as start of statement.
self.assertEqual(
    self.repo.last_response.json()['id'],
    self.__class__.__name__,
    set(self.repo.last_response.json()))
# No: hanging arguments are used.
self.assertEqual(self.repo.last_response.json()['id'],
                 self.__class__.__name__,
                 set(self.repo.last_response.json()))
# No: multiple arguments on same line
self.assertEqual(
    self.repo.last_response.json()['id'], self.__class__.__name__
)
```